plugin/technologymodel.cpp

```diff
@@ -267,25 +283,14 @@ void TechnologyModel::updateServiceList()
 
     const int num_old = m_services.count();
 
-    foreach(const NetworkService *s, m_services) {
-        disconnect(s, SIGNAL(destroyed(QObject*)),
-                   this, SLOT(networkServiceDestroyed(QObject*)));
-    }
-
     const QVector<NetworkService *> new_services =
-        (m_filter == SavedServices) ? m_manager->getSavedServices(m_techname) :
-        (m_filter == AvailableServices) ? m_manager->getAvailableServices(m_techname) :
-        m_manager->getServices(m_techname);
+        (m_filter == SavedServices) ? m_manager->getSavedServices(m_techname)
```

Comment:
If this is reformatted why not remove the use of ternary altogether in favor of more clearer, easy to read solution approach?

Reply:
Not that bad when it's according to coding contentions. Borderline which is better.

Reply:
Plain `if() ... else if() ... else` is at least for me easy to read with one glance. This ternarystuff is not.